tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -420,6 +423,7 @@ generate_qemu_options() {
 
 	# Support Linux AIO (native)
 	qemu_options+=(size:--enable-linux-aio)
+	qemu_options+=(size:--enable-linux-io-uring)
```

Comment:
@zvonkok - it looks like this belongs to some other patch...

Reply:
We need io-uring support building without explicitly stating it fails to build 9.1.1 somehow.